esmvaltool/utils/cmorizers/obs/cmor_config/Landschutzer2014.yml

```diff
@@ -0,0 +1,42 @@
+---
+# project at hand
+proj:
+  dataset: 'Landschutzer2014'
+  version: 'CDIAC_G05'
+  realm: 'clim'
+  field: 'TO2M'
+  frequency: {
+        'fgco2': 'Omon',
+        'spco2': 'Omon',
+        'dpco2': 'Omon',
+    }
+  metadata_attributes:
+    tier: '2'
+    source: 'CRESCENDO WP5 (CMCC)'
+    comment: 'cmorized for ESMValTool v2'
+    CMORconventions: 'CF/CMOR3'
+    CMORcreated: ''
+
+# Match CMOR variables with input file one(s)
+VAR_TO_CMOR:
+  fgco2: 'fgco2_smoothed' 
+  spco2: 'spco2_smoothed' 
+  dpco2: 'dco2_smoothed' 
+
+# specific raw file names (implicitly assumed to be .nc)
+VAR_TO_FILENAME:
+  fgco2: 'spco2_1998-2011_ETH_SOM-FFN_CDIAC_G05'
+  spco2: 'spco2_1998-2011_ETH_SOM-FFN_CDIAC_G05'
+  dpco2: 'spco2_1998-2011_ETH_SOM-FFN_CDIAC_G05'
+
+# cmor standard names
+STANDARD_NAMES:
+  fgco2: 'surface_downward_mass_flux_of_carbon_dioxide_expressed_as_carbon'
+  spco2: 'surface_partial_pressure_of_carbon_dioxide_in_sea_water'
+  dpco2: 'surface_carbon_dioxide_partial_pressure_difference_between_sea_water_and_air'
+
+# cmor long names
```

Comment:
This should be read from the cmor tables, specifying them here is unnessary and error prone

Reply:
again, these are extracted from the raw obs files and can have a whole jungle of names

Reply:
If there is no associated cmor table, the result from cmorization cannot be used in esmvaltool anyway

Reply:
?? dude, you don't get the problem at hand - what CMOR table? These come from the __RAW__ obs files that are crazy and need to be set by the person who wants to cmorize stuff

Reply:
Have another look please: this comment is on the cmor standard_name and long_name that will be assigned the cmorized variable. The writer of the cmorization script should not have to provide these.

Reply:
ow my bad sorries :roll_eyes: OK but the mapping to CMOR tables has to be done via at least one CMOR parameter and what would the best one? I reckon standard_name but that could be tricky to chose. On top of everything else the user may actually decide another CMOR variable may be better suited to represent the one in the raw file